src/data/docs/abilities.md

```diff
@@ -0,0 +1,578 @@
+---
+title: Abilities in Unison
+description: Unison provides a convenient feature called _abilities_ which lets you use the same ordinary Unison syntax for programs that do I/O, stream processing, parsing, distributed computation, and lots more.
+---
+
+# Abilities in Unison
+
+Unison provides a convenient feature called _abilities_ which lets you use the same ordinary Unison syntax for programs that do (asynchronous) I/O, stream processing, exception handling, parsing, distributed computation, and lots more. This tutorial walks the basics of how you'll interact with abilities and how they are typechecked, using `IO` as an example, then we'll cover how you can create and use new abilities for yourself. There are some (extremely fun and optional) exercises you can do along the way, and answers at the end of this document!
```

Comment:
This has been reworked quite a bit in the latest version, which just starts with an example right away rather than abstractly trying to explain what abilities are and why they are useful.